easybuild/easyconfigs/a/avogadrolibs/avogadrolibs-1.95.1-gompi-2021a.eb

```diff
@@ -0,0 +1,72 @@
+easyblock = 'CMakeMake'
+
+name = 'avogadrolibs'
+version = '1.95.1'
+
+homepage = 'https://two.avogadro.cc/'
+description = """Avogadro is an advanced molecule editor and visualizer
+designed for cross-platform use in computational chemistry, molecular
+modeling, bioinformatics, materials science, and related areas. It
+offers flexible high quality rendering and a powerful plugin
+architecture.
+
+This is avogadrolibs for Avogadro2.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
+
+source_urls = ['https://github.com/OpenChemistry/avogadrolibs/archive/']
+sources = ['%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version)s_fix_VTK_component_names.patch',
+]
+checksums = [
+    '52817eebfc0cf700e5f6c0c4455b460618ec0dd7cb4ede5d8fabc5a2d73a7816',  # 1.95.1.tar.gz
+    # avogadrolibs-1.95.1_fix_VTK_component_names.patch
+    '723d54060cdef141d2c9f8586a0c04b64850b8c407f087b3ad75cf1c8723bf05',
+]
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('pybind11', '2.6.2'),
+    ('zlib', '1.2.11'),
+    ('Eigen', '3.3.9'),
+    ('spglib', '1.16.2'),
+    ('mmtf-cpp', '1.0.0'),
+    ('libmsym', '0.2.3'),
+    ('glew', '2.1.0'),
+    ('HDF5', '1.10.7'),
+    ('X11', '20210518'),
+    ('Mesa', '21.1.1'),
+    ('Qt5', '5.15.2'),
+    # ('VTK', '9.0.1'), # Current VTK lacks the GUISupportQt and RenderingContextOpenGL2 components
```

Comment:
Current as in VTK version 9.0.1, or how we install VTK?

Reply:
When trying with VTK (which is foss so have to up this one) I get:
```
CMake Error at avogadro/vtk/CMakeLists.txt:14 (find_package):
  Found package configuration file:

    /hpc2n/eb/software/VTK/9.0.1-foss-2021a/lib/cmake/vtk-9.0/vtk-config.cmake

  but it set VTK_FOUND to FALSE so package "VTK" is considered to be NOT
  FOUND.  Reason given by package:

  Could not find the VTK package with the following required components:
  GUISupportQt, RenderingContextOpenGL2.
```